specification/authorization/resource-manager/Microsoft.Authorization/preview/2024-02-01-preview/RoleManagementPolicyAssignment.json

```diff
@@ -0,0 +1,393 @@
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "GetRoleManagementPolicyAssignmentByScope": {
+            "$ref": "./examples/GetRoleManagementPolicyAssignmentByScope.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "RoleManagementPolicyAssignment": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The role management policy Id."
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The role management policy name."
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The role management policy type."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/RoleManagementPolicyAssignmentProperties",
+          "description": "Role management policy properties."
+        }
+      },
+      "type": "object",
+      "description": "Role management policy"
+    },
+    "RoleManagementPolicyAssignmentProperties": {
+      "properties": {
+        "scope": {
+          "type": "string",
+          "description": "The role management policy scope."
+        },
+        "roleDefinitionId": {
+          "type": "string",
+          "description": "The role definition of management policy assignment."
+        },
+        "policyId": {
+          "type": "string",
+          "description": "The policy id role management policy assignment."
+        },
+        "effectiveRules": {
+          "type": "array",
+          "items": {
+            "$ref": "./common-types.json#/definitions/RoleManagementPolicyRule"
+          },
+          "readOnly": true,
+          "description": "The readonly computed rule applied to the policy."
+        },
+        "policyAssignmentProperties": {
+          "readOnly": true,
+          "type": "object",
+          "description": "Additional properties of scope, role definition and policy",
+          "$ref": "#/definitions/PolicyAssignmentProperties"
+        }
+      },
+      "type": "object",
+      "description": "Role management policy assignment properties with scope."
+    },
+    "RoleManagementPolicyAssignmentListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RoleManagementPolicyAssignment"
+          },
+          "description": "Role management policy assignment list."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to use for getting the next set of results."
+        }
+      },
+      "type": "object",
+      "description": "Role management policy assignment list operation result."
+    },
+    "PolicyAssignmentProperties": {
+      "properties": {
+        "scope": {
```

Comment:
>scope [](http://example.com/codeflow?start=9&length=5)

I'm finding this definition confusing. Looking at the property names and description, it seems more like a link/reference to a single resource rather than a scope. You need to model this more clearly so it's obvious which it is.